scripts/check-dialyzer.escript

```diff
@@ -38,8 +37,7 @@ print_help() ->
     getopt:usage(option_spec_list(), "ERL_LIBS=deps/:core/:applications/ " ++ Script ++ " .kazoo.plt [args] [file.beam | path/ebin/ ...]"),
     halt(1).
 
-handle(_KazooPLT, _Options, []) ->
-    print_help();
+handle(_KazooPLT, _Options, []) -> 'ok';
```

Comment:
But this clause is needed to handle when no files were passed AND there was a flag such as `--bulk` or `--hard` passed. And it will handle the L13 clause as well.